.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
not having this here tripped me up a lot in the beginning, since prettier is a pretty common default config